backend/backend/Controllers/UsersController.cs

```diff
@@ -130,5 +131,73 @@ public ActionResult Verify(Guid id)
 
             return Ok();
         }
+
+        [HttpPost, Route("forgot-password"), AllowAnonymous]
+        public ActionResult ForgotPassword(string email)
+        {
+            if (string.IsNullOrWhiteSpace(email))
+            {
+                return BadRequest();
+            }
+
+            var user = _db.Users.FirstOrDefault(x => x.Email == email);
+            if (user == null)
+            {
+                return Ok();
```

Comment:
shouldn't we return 404 here?

Reply:
I believe that this was done for security reasons. If you return anything else than 200 an Evil Guy from the Dark Web would just be able to know if a person has registered in our system just by checking the response code. Or maybe Mr. @kibartas  had something else in his mind